### ISSUE TITLE
Per-user key management

### ISSUE THREAD

**Issue by ikonovalenko** [skavanagh/KeyBox#106]
Hello,

Is it possible to implement an ability to create per-user keys like:

The system is added with root privilegies.
When user generates key there is an option to select for which use on the remote system the key will be added.
If user selects "user1234" as an user, the KeyBox system will add key to /home/user1234/.ssh folder.

**Comment by skavanagh** [comment]
Currently it's not per system, it's added to all the systems for the profile that is selected.

**Comment by ikonovalenko** [comment]
Yeah, I know how it works currently. I meant is it possible to implement it in one of next releases?
